### ISSUE TITLE
Allowing users to specify error message text for blank fields.

### ISSUE THREAD

**Issue by Mattcessability** [Tanaguru/contact-form-7#2]
Hey there,

I'm using your accessibility fork of CF7 on our website (https://modalityco.com), and it does a really great job and is significantly better than the regular CF7. Really awesome you created this.

It mostly passes WCAG, but it falls short in error messages. Namely:

•	Error messages do appear under blank form field in error, but they are generic “Please fill out this field” messages. Ideally, we’d want error messages to be a little more specific and explicitly mention which form field is in error. This is a 3.3.1 Error identification issue in WCAG.
•	The error messages also don’t come with an icon, or allow you to put one in. This creates a 1.4.1 Use of Colour failure in WCAG because the red error message text doesn’t stand out well, especially for many colour-blind individuals.

There used to be a [custom error messages plugin for CF7](https://wordpress.org/plugins/cf7-custom-error-messages/?ref=commonninja), but it’s sadly been taken down, presumably because the developer didn’t have the time to keep maintaining it.

It would be awesome if we could have these features added to this fork, as I believe it would allow the plugin to be at least WCAG conformant - providing a viable alternative for the paid Gravity Forms.

Cheers,
Matt.
